index.bs

```diff
@@ -4152,6 +4124,11 @@ Constructors</h4>
 <dl dfn-type=constructor dfn-for="AnalyserNode">
 	: <dfn>AnalyserNode(context, options)</dfn>
 	::
+		When the constructor is called with a {{BaseAudioContext}} <var>c</var> and
```

Comment:
I didn't check everywhere, but it looks like this paragraph is the same for every node.  Perhaps a bikeshed macro would be useful here so that there's only one place to fix things and it will get updated everywhere.

Reply:
It is exactly the same, I've factored it out in a macro.